trending-topic-campaigns/activiti-cloud-connectors-3rd-party/src/main/java/org/activiti/cloud/connectors/external/ShoutCallStrategy.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2017 Alfresco, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.activiti.cloud.connectors.external;
+
+import org.activiti.cloud.connectors.external.model.Shout;
+
+public interface ShoutCallStrategy {
```

Comment:
@erdemedeiros I have some issues with the name: ShoutCallStrategy -> can we just call this ShoutService?

Reply:
Or maybe ShoutServiceClient.. so we have one SLAShoutServiceClient and one Simple/Basic ShoutServiceClient

Reply:
Yeah, it makes sense. I'll rename it.